src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -33,7 +35,7 @@ const NO_OPERATION_SUCCESS = {
 const identifyPayloadBuilder = (event) => {
   const message = new Message(INTEGERATION);
   message.setEventType(EventType.IDENTIFY);
-  message.setPropertiesV2(event, MAPPING_CATEGORIES[EventType.IDENTIFY]);
+  message.setPropertiesV2(event, identifyMappingJSON);
```

Comment:
I don't see any difference in the two `identify` mapping configuration

Reply:
Yes, just wanted there should be least dependency on old v0 source code, so as to allow granular control w.r.t new v1 source transformations without any concerns for supporting backwards compatibility.